docs/flutter.md

```diff
@@ -4,15 +4,41 @@ Official RICOH [demo-flutter](https://github.com/ricohapi/theta-client/tree/main
 
 ## Community Video Tutorials
 
-* [theta-client new Flutter App from the beginning - blank editor to Android app](https://youtu.be/EAwT2j0x5VU)
-* [Equirectangular to 360 Image in Flutter  with Panorama Package](https://youtu.be/9hWUU6G3Ank) - published February 21, 2023
-* [RICOH theta-client Flutter Build on Windows](https://youtu.be/dJV5s46xFA0) - published Feb 8, 2023.
+* [theta-client new Flutter App from the beginning - blank editor to Android
+app](https://youtu.be/EAwT2j0x5VU)
```

Comment:
Suggest adding date published to match next two videos - "published March 2, 2023"

Reply:
OK.  I made this change on the craig-android branch

Reply:
Oh, nice maneuver. It's live, so it has worked.